Update config_rb_server_optional_settings.rst (#2019)
chef/chef-web-docs: chef_master/source/config_rb_server_optional_settings.rst

   The maximum connection duration (in seconds) for the HTTP connection pool that is used by the data collector. Default value: `"{70, sec}"`.
``data_collector['ibrowse_options']``
   An array of comma-separated key-value pairs of ibrowse options for the HTTP connection pool that is used by the data collector. Default value: `"[{connect_timeout, 10000}]"`.
``data_collector['health_check']``

>    A boolean that controls whether the data collector health is included in the overall health at the ``_status`` endpoint. This setting when set to ``false`` is useful to prevent perfectly healthy front end Chef cluster members being reported as failed and removed from a load balancer pool when the ``data_collector['root_url']`` cannot be reached. Default value: ``true``.

Reviewer: ```suggestion
   A boolean that controls whether the data collector health is included in the overall health at the ``_status`` endpoint. When set to ``true``, Chef Infra Server will report that healthy front end Chef HA cluster members have failed when the `data_collector['root_url']`` cannot be reached. As a result, the load balancer will remove those members from the load balancer pool. Default value: ``true``.
```

Author: Looks great, ship it ;-)
